cinema/views.py

```diff
@@ -21,24 +23,45 @@
 )
 
 
-class GenreViewSet(viewsets.ModelViewSet):
+class GenreViewSet(
+    viewsets.GenericViewSet,
+    mixins.ListModelMixin,
+    mixins.CreateModelMixin
+):
     queryset = Genre.objects.all()
     serializer_class = GenreSerializer
+    authentication_classes = (TokenAuthentication,)
```

Comment:
why do you override auth classes by default auth class when you set default one in settings?

Reply:
fix everywhere